netdata-installer.sh

```diff
@@ -118,6 +118,13 @@ download_go() {
 # make sure we save all commands we run
 run_logfile="netdata-installer.log"
 
+# set default make options
+if [ -z "${MAKEOPTS}" ]; then
+  MAKEOPTS="-j$(find_processors)"
+elif echo "${MAKEOPTS}" | grep -vqF "-j"; then
```

Comment:
Should we add `--` before the `"-j"` ? In a test I get: `grep: invalid option -- 'j'` when I try e.g. `MAKEOPTS="-j3" ./netdata-installer.sh ....`

Reply:
Ah, yeah, we do need that...

Reply:
Though for `grep`, it’s `-e` for the pattern. `--` is used to mark the start of a file list to search through.